examples/emberjs/todomvc/app/templates/application.hbs

```diff
@@ -1,24 +1,29 @@
-<section id="todoapp">
-  <header id="header">
+<section class="todoapp">
+  <header class="header">
     <h1>todos</h1>
-    <input type="text" id="new-todo" onkeydown={{action 'createTodo'}} placeholder="What needs to be done?" autofocus>
+    <input
+      class="new-todo"
+      onkeydown={{action "createTodo"}}
+      placeholder="What needs to be done?"
+      autofocus
+    >
   </header>
-    {{outlet}}
-    {{#if (gt model.length 0)}}
-      <footer id="footer">
-        <span id="todo-count"><strong>{{remaining.length}}</strong> {{pluralize 'item' remaining.length}} left</span>
-        <ul id="filters">
-          <li>{{#link-to "index" activeClass="selected"}}All{{/link-to}}</li>
-          <li>{{#link-to "active" activeClass="selected"}}Active{{/link-to}}</li>
-          <li>{{#link-to "completed" activeClass="selected"}}Completed{{/link-to}}</li>
-        </ul>
-        {{#if completed.length}}
-          <button id="clear-completed" onclick={{action 'clearCompleted'}}>Clear completed</button>
-        {{/if}}
-      </footer>
-    {{/if}}
+  {{outlet}}
+  {{#if (gt model.length 0)}}
+    <footer class="footer">
+      <span class="todo-count"><strong>{{remaining.length}}</strong> {{pluralize "item" remaining.length}} left</span>
```

Comment:
I think all the variables and such should be prefixed with 'this'

Also, should we extract components?

Reply:
What components did you have in mind?

Reply:
one for footer, and header :)

Reply:
Decided against this for now. I think this MR should only about updating ember and use the newest TodoMVC template.